andrew_bird/week_02/knuthWebsite/kt.css

```diff
@@ -0,0 +1,83 @@
+
+div{
+	
+	clear: both;
+}
+
+header {
+border-bottom: 5px double black; }
+
+nav {
+float: left;
+width: 20%;
+clear: left;
+min-height: 600px; border-right: 1px solid black;
+}
+
+section {
+float: left; width: 75%; padding-left: 1em;
+}
+
+article {
+float: left; width: 75%; padding-left: 1em;
```

Comment:
Try to avoid floats as it causes lot's of mess on pages. And try to stick in a file to one convention of width, for example. If you are using px, keep using them for everything(instead of px, %, em, rem). Best to use are em and rem as they helps to make pages more flexible and if you want to change sizes you can do it in one place.